air/src/preparation_step/preparation.rs

```diff
@@ -134,14 +139,16 @@ fn make_exec_ctx(
     signature_store: SignatureStore,
     run_parameters: &RunParameters,
 ) -> PreparationResult<ExecutionCtx<'static>> {
-    let call_results = measure!(
+    let call_results_js = measure!(
```

Comment:
what is the meaning of the `js`suffix here?

Reply:
JS-compatible representation of call results.